test/util.ml

```diff
@@ -9,5 +9,5 @@ let with_test_file path contents f =
 let with_test_dir path f =
   Sys.mkdir (Path.to_string path) 0o777;
   let res = f path in
-  Sys.rmdir (Path.to_string path);
```

Comment:
`Sys.rmdir` won't remove a directory if it has files in it. Doing `rm -r` seems a lot easier than writing our own.